packages/sample/package.json

```diff
@@ -6,8 +6,17 @@
     "private": true,
     "type": "module",
     "dependencies": {
+        "@rev-robotics/control-hub": "*",
         "@rev-robotics/expansion-hub": "*",
-        "commander": "^10.0.1"
+        "ws": "8.13.0",
+        "commander": "^10.0.1",
+        "@u4/adbkit": "^4.1.19",
+        "get-port": "^6.1.2"
+    },
+    "devDependencies": {
+        "@types/ws": "^8.5.4",
+        "@types/node-forge": "^1.3.2",
+        "@types/debug": "^4.1.8"
```

Comment:
Why are these needed?

Reply:
adbkit needs the second and third, ws needs the first.